go/stalin/stalin_test.go

```diff
@@ -1,59 +0,0 @@
-package stalin
-
-import (
-	"reflect"
-	"sort"
-	"testing"
-)
-
-func TestSort(t *testing.T) {
-	golden := []struct {
-		Got, Want []int
-	}{
-		{
-			[]int{1, 2, 10, 3, 2, 4, 15, 6, 30, 20},
-			[]int{1, 2, 10, 15, 30},
-		},
-	}
-
-	for _, gold := range golden {
-		list := make([]int, len(gold.Got))
-		copy(list, gold.Got)
-		n := Sort(list)
-
-		got := list[:n]
-		if !reflect.DeepEqual(got, gold.Want) {
-			t.Errorf("%v: got %v, want %v", gold.Got, got, gold.Want)
-		}
-
-		remaining := list[n:]
-		if !reflect.DeepEqual(remaining, make([]int, len(remaining))) {
-			t.Errorf("%v: remaining not zero: %v", gold.Got, remaining)
-		}
-	}
-}
-
-func BenchmarkSort(b *testing.B) {
```

Comment:
can we migrate this benchmark?

Reply:
Done! :) If you're interested in the difference
```
$ benchcmp old.txt new.txt                  
benchmark                        old ns/op     new ns/op     delta
BenchmarkSortInts/stalin-8       17.1          14.0          -18.13%
```